src/mame/machine/bublbobl.cpp

```diff
@@ -86,41 +143,41 @@ void bublbobl_state::device_timer(emu_timer &timer, device_timer_id id, int para
 	}
 }
 
-
-WRITE8_MEMBER(bublbobl_state::bublbobl_sound_command_w)
-{
-	m_soundlatch->write(space, offset, data);
-	synchronize(TIMER_NMI, data);
-}
-
 WRITE8_MEMBER(bublbobl_state::bublbobl_sh_nmi_disable_w)
 {
-	m_sound_nmi_enable = 0;
+	m_sound_nmi_enable = false;
```

Comment:
You can definitely do this with an input merger and cascaded devcb.  Please think about it and do it properly.